src/actions/ui.js

```diff
@@ -38,6 +38,8 @@ export function setActiveSearch(activeSearch?: ActiveSearchType) {
       return;
     }
 
+    dispatch({ type: "CLOSE_QUICK_OPEN" });
```

Comment:
lets add an if statement so we only fire it if it's open.

we can also add a unit test here where we open the modal and then fire SAS